go.mod

```diff
@@ -74,6 +74,7 @@ require (
 	google.golang.org/protobuf v1.25.0 // indirect
 	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
 	gopkg.in/ini.v1 v1.57.0 // indirect
+	gopkg.in/src-d/go-log.v1 v1.0.1
```

Comment:
I am not sure this change is needed

Reply:
Oh sorry, I guess this came with the merge with the master. I'll remove it.